python/src/GeneralizedExtremeValueFactory_doc.i.in

```diff
@@ -11,404 +9,410 @@ DistributionFactory, GeneralizedExtremeValue, FrechetFactory, GumbelFactory, Wei
+The result class produced by the method provides:
+
+- the GEV distribution associated to :math:`(\hat{\mu}, \hat{\sigma}, \hat{\xi})`,
+- the asymptotic distribution of :math:`(\hat{\mu}, \hat{\sigma}, \hat{\xi})`.
 
 Parameters
 ----------
-sample : :class:`~openturns.Sample`
-    Data
+sampleM : 2-d sequence of float
+    Block maxima  grouped in a sample of size :math:`n` and dimension :math:`R`.
+r : int, :math:`1 \leq r \leq R`, optional
+    Number of order statistics taken into account among the :math:`R` stored ones.
+    
+    By default, :math:`r=0` which means that all the maxima are used.
 
 Returns
 -------
-result : :class:`~openturns.ProfileLikelihoodResult`
-    The associated results"
+result : :class:`~openturns.DistributionFactoryLikelihoodResult`
+    The result class."
 
 // ----------------------------------------------------------------------------
 
-%feature("docstring") OT::GeneralizedExtremeValueFactory::buildRMaxima
-"Estimate the distribution with R maxima method.
-
-Let :math:`M_i^{(r)} = (z_i^{(1)}, \hdots, z_i^{(r)})` the :math:`r` largest values
-of block :math:`i`, and sorted by decreasing order.
+%feature("docstring") OT::GeneralizedExtremeValueFactory::buildTimeVarying
+"Estimate a non stationary GEV.
 
-The parameters are estimated thanks to a numerical optimization of the likelihood:
+We consider a non stationary GEV model to describe the distribution of :math:`Z_t`:
 
 .. math::
 
-    L(\mu, \sigma, \xi) = \prod_{i=1}^m \Biggl( \exp \Biggl\{ - \biggl[ 1 + \xi \Bigl( \frac{z_i^{(r)} - \mu }{\sigma} \Bigr) \biggr]^{-1/\xi} \Biggr\} \prod_{k=1}^r \sigma^{-1} \biggl[ 1 + \xi \Bigl( \frac{z_i^{(k)} - \mu }{\sigma} \Bigr) \biggr]^{-1/\xi-1} \Biggr)
+    Z_t \sim \mbox{GEV}(\mu(t), \sigma(t), \xi(t))
+
+We have the values of :math:`Z_t` on the time stamps :math:`(t_1, \dots, t_n)`.
 
-if :math:`1+\xi \left( \frac{z_i^{(k)} - \mu}{\sigma} \right) > 0 \forall i \forall k`, and by:
+For numerical reasons, it is recommended to normalize the time stamps. OpenTURNS applies the following mapping:
 
 .. math::
 
-    L(\mu, \sigma, \xi) = \prod_{i=1}^m \Biggl( \exp \Biggl\{ - \exp \biggl[ - \Bigl( \frac{z_i^{(r)} - \mu }{\sigma} \Bigr) \biggr] \Biggr\} \prod_{k=1}^r \sigma^{-1} \exp \biggl[ - \Bigl( \frac{z_i^{(k)} - \mu }{\sigma} \Bigr) \biggr] \Biggr)
+    \tau(t) = \dfrac{t-c}{d}
 
-if :math:`\xi = 0`.
+and with three ways of defining :math:`(c,d)`:
 
-Parameters
-----------
-sample : :class:`~openturns.Sample`
-    Sample of size :math:`M` (the number of independent samples),
-    and dimension :math:`R` (the number of maxima per sample)
-    Each record of :math:`R` maxima are sorted in decreasing order
-r : int, default=0 (sets :math:`r=R` to use all maxima)
-    Number of maximas taken into account
+- the *CenterReduce* method where :math:`c = \dfrac{1}{n} \sum_{i=1}^n t_i` is the mean time stamps
+  and :math:`d = \sqrt{\dfrac{1}{n} \sum_{i=1}^n (t_i-c)^2}` is the standard deviation of the time stamps;
```

Comment:
The layout of the equations could be improved here:

![image](https://github.com/openturns/openturns/assets/31351465/4bc74dd6-e4d3-4e59-9e9a-af07e97615ba)

I suggest to used centered equations:
```suggestion
# - the "CenterReduce" method where
#
# .. math::
# 
#     c = \dfrac{1}{n} \sum_{i=1}^n t_i
#
#   is the mean time stamps and:
#
# .. math:
#
#     d = \sqrt{\dfrac{1}{n} \sum_{i=1}^n (t_i-c)^2}
#
#    is the standard deviation of the time stamps;
```
I do not know the conditions to ensure that the intermediate text stays indented.

Reply:
ok